docs/data/rpc/api-reference/methods/getLedgerEntries.mdx

```diff
@@ -48,291 +147,179 @@ print(
-This functionality is included in the JavaScript [`stellar-sdk`](https://www.npmjs.com/package/stellar-sdk) package as `rpc.Server.getAccount(address)`.
-
-:::
-
-Accounts are stored as ledger entries, so we can use this method to look up an account along with it's current sequence number.
-
-```js
-import { xdr, Keypair } from '@stellar/stellar-sdk'
-
-const getLedgerKeyAccount = (address) => {
-  const ledgerKey = xdr.LedgerKey.account(
-    new xdr.LedgerKeyAccount({
-      accountId: Keypair.fromPublicKey(address).xdrPublicKey(),
-    })
-  )
-  return ledgerKey.toXDR('base64')
-}
-
-console.log(getLedgerKeyAccount(
-  'GCU5YE6IVBOEZ5LUU5N2NB55VPB5YZNFERT65SSTVXTNMS7IEQWXKBM2'
-))
-
-# OUTPUT: AAAAAAAAAACp3BPIqFxM9XSnW6aHvavD3GWlJGfuylOt5tZL6CQtdQ==
-```
-
-We then take our output from this function, and use it as the element in the `keys` array parameter in our call to the `getLedgerEntries` method.
-
-```json
-{
-  "jsonrpc": "2.0",
-  "id": 8675309,
-  "method": "getLedgerEntries",
-  "params": {
-    "keys": ["AAAAAAAAAACp3BPIqFxM9XSnW6aHvavD3GWlJGfuylOt5tZL6CQtdQ=="]
-  }
-}
-```
-
-And the response we get contains the `LedgerEntryData` with the current information about this account.
-
-```json
-{
-  "jsonrpc": "2.0",
-  "id": 8675309,
-  "result": {
-    "entries": [
-      {
-        "key": "AAAAAAAAAACp3BPIqFxM9XSnW6aHvavD3GWlJGfuylOt5tZL6CQtdQ==",
-        "xdr": "AAAAAAAAAACp3BPIqFxM9XSnW6aHvavD3GWlJGfuylOt5tZL6CQtdQAAABdIdugAAAWpygAAAAAAAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAAAAAAA",
-        "lastModifiedLedgerSeq": "164303"
-      }
-    ],
-    "latestLedger": 246819
-  }
-}
-```
-
-We can then parse this result as an `xdr.LedgerEntryData` type.
-
-```js
-const parsed = xdr.LedgerEntryData.fromXDR(
-  "AAAAAAAAAACp3BPIqFxM9XSnW6aHvavD3GWlJGfuylOt5tZL6CQtdQAAABdIdugAAAWpygAAAAAAAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAAAAAAA",
-  "base64",
+const ledgerKey = getLedgerKeySymbol(
+  "CCPYZFKEAXHHS5VVW5J45TOU7S2EODJ7TZNJIA5LKDVL3PESCES6FNCI",
+  "COUNTER",
 );
-console.log(parsed);
 ```
 
-### Requesting a Contract's Wasm Code
+</CodeExample>
 
-This can be a bit tricky to wrap your head around, but the conventions do make sense once you let it sink in.
+### Contract Wasm Code
 
-In the previous examples, the `COUNTER` _key_ was used as a `LedgerKey` while the incremented _value_ was stored in a **`LedgerEntry`**. "Ledger Entry" is the relevant term to keep in mind during this discussion. That `LedgerEntry` was stored on the Stellar ledger, and was associated with a corresponding `LedgerKey`. `LedgerKey: LedgerEntry` works the same way you would think of almost any `key: value` storage system.
+To understand this, we need a handle on how smart contract deployment works:
 
-#### How Soroban Contract Deployment Works
+- When you deploy a contract, first the code is "installed" (i.e. uploaded onto the blockchain), creating a `LedgerEntry` with the Wasm byte-code that can be uniquely identified by its hash (that is, the hash of the uploaded code itself).
```

Comment:
I think you should try to use the same terminology / description as https://developers-pr1060.previews.kube001.services.stellar-ops.com/docs/build/smart-contracts/getting-started/deploy-increment-contract . That page states that deployment is a two step process with first step being uploading / installing the code and the second step being instantiation.

In this snippet, you refer to the second step as deployment which might be confusing since the other page defines it as creation / instantiation.

Reply:
Good call, updated :+1: